frontend/src/app/components/Contact/Contact.tsx

```diff
@@ -0,0 +1,54 @@
+import styles from "@/app/components/Sidebar/sidebar.module.scss";
+import matter from "gray-matter";
```

Comment:
If you go for the suggestion in the next comment below, you'll need this:

```suggestion
import matter, { GrayMatterFile } from "gray-matter";
```

Reply:
I will not commit any of the suggestions, as I wrote I have been working on a fix and committed it that fixes the typing and path issues.